### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gendeld**

### CODE REVIEW

File: .storybook/addons.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
 
 import '@storybook/addon-actions/register';
-import '@storybook/addon-links/register';
+// import '@storybook/addon-links/register';
```
Comment (shlomitc) [inline .storybook/addons.js:4]: Remove?

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "start": "npm run storybook",
     "pretest": "yoshi lint && yoshi build",
     "build": "yoshi lint && yoshi build && build-storybook",
-    "test": "yoshi test",
+    "test": "yoshi test --passWithNoTests",
```
Comment (shlomitc) [inline package.json:21]: If there are not tests - you can simply remove it

File: package.json [hunk 3]
```diff
@@ -29,44 +29,41 @@
...
   "devDependencies": {
-    "@storybook/react": "^3.1.7",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
+    "@storybook/addon-actions": "^4.1.18",
+    "@storybook/addon-links": "^6.0.26",
+    "@storybook/react": "^4.1.18",
     "chai": "^3.5.0",
     "copy-to-clipboard": "^3.0.8",
```
Comment (shlomitc) [inline package.json:36]: Are you sure this is needed?

Comment (gendeld) [reply package.json:36]: It failed without it

Comment (gendeld) [reply package.json:34]: I remove addon-links

Comment (shlomitc) [reply package.json:34]: really strange. Where is the error coming from?

**Review by EgoziE — APPROVED** [review]

**Pull request merged**